### PR TITLE
LPS-185226 | Automation for Automation DatasetsFields

### PR DESCRIPTION
**Pull request by abhnerramos**
**Description**
Add 4 automations for DatasetsFields testcase

**JIRA Tickets**
DatasetsFields#CanAddField ([LPS-185226](https://issues.liferay.com/browse/LPS-185226))
DatasetsFields#CanSearchFieldOnAddFieldsModal ([LPS-185227](https://issues.liferay.com/browse/LPS-185227))
DatasetsFields#CannotAddFieldAfterUncheckingIt ([LPS-185228](https://issues.liferay.com/browse/LPS-185228))
DatasetsFields#CanChangeFieldOrder ([LPS-185229](https://issues.liferay.com/browse/LPS-185229))

### CODE REVIEW

**Comment by abhnerramos** [comment]
ci:test:sf

**Comment by abhnerramos** [comment]
ci:test:frontend-data-set

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/DatasetsFields.testcase [hunk 1]
```diff
@@ -124,12 +139,48 @@ definition {
...
 
-		// TODO LPS-185229 CanChangeFieldOrder pending implementation
+		task ("And checks id and label fields") {
+			Check.checkNotVisible(
+				checkboxName = "id",
+				locator1 = "Checkbox#ANY_CHECKBOX");
 
+			Check.checkNotVisible(
+				checkboxName = "label",
+				locator1 = "Checkbox#ANY_CHECKBOX");
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/DatasetsFields.testcase:155]: I would use the macro here too and in the other tests that make this check

Comment (abhnerramos) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/DatasetsFields.testcase:155]: Oh yes, nice! Will work on that

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/DatasetsFields.testcase [hunk 2]
```diff
@@ -124,12 +139,48 @@ definition {
...
+				locator1 = "FrontendDataSetAdmin#FIELDS_DRAG_DOTS_ICON",
+				locator2 = "FrontendDataSetAdmin#FIELDS_TABLE");
+
+			FrontendDataSetAdmin.assertFieldsOnTable(
+				key_position = 1,
+				keyName = "label");
+
+			FrontendDataSetAdmin.assertFieldsOnTable(
+				key_position = 2,
+				keyName = "id");
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/DatasetsFields.testcase:182]: Great!!! :D

**Review by CarlosBrichete — APPROVED** [review]

**Comment by abhnerramos** [comment]
ci:test:frontend-data-set

**Pull request closed without merging**